imblearn/over_sampling/tests/test_rose.py

```diff
@@ -0,0 +1,59 @@
+"""Test the module ROSE."""
+# Authors: Andrea Lorenzon <andrelorenzon@gmail.com>
+# License: MIT
+
+import numpy as np
+
+from imblearn.over_sampling import ROSE
+
+
+def test_testunit():
+    """Check test unit"""
+    return True
+
+
+def test_random_state():
+    """Check randomState()"""
+    assert np.random.RandomState(42)
+
+
+def test_instance():
+    """Check ROSE instantiation"""
+    rose = ROSE()
+    assert id(rose)
```

Comment:
These test cases do not appear to be checking for behavior of the `ROSE` class.

Reply:
They were used only during development. I removed them.